localstack/cli/localstack.py

```diff
@@ -37,17 +47,38 @@ def _setup_cli_debug():
     setup_logging()
 
 
+def _publish_cmd_as_analytics_event(cmd: str):
```

Comment:
could we maybe move this method into `localstack.utils.analytics`, then we could keep the cli package mostly clean of analytics imports.